packages/map/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@next/eslint-plugin-next": "^11.1.2",
+    "chokidar": "^2.1.8",
```

Comment:
You added this new dependency but it is not used anywhere that I can see. 
Is it really needed, or can it be removed?

Reply:
This package was added by mistake. Thank you